scripts/cms5/merge-ups.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+// run 3 days offset from when a weekly CI run is, otherwise run on a random day of the week
+$runOnDay = predictable_random_int(6);
+// run at the same hour as the weekly CI run, otherwise run at a random hour of the day
+$runOnHour = predictable_random_int(23);
+// run at the same minute as the weekly CI run, otherwise run at a random minute of the hour
+// rounded to 5 minutes
```

Comment:
These comments are misplaced, since the 3 days offset and other "same as CI run" things happen later.
```suggestion
// run on a random day of the week
$runOnDay = predictable_random_int(6);
// run at a random hour of the day
$runOnHour = predictable_random_int(23);
// run at a random minute of the hour rounded to 5 minutes
```

Reply:
Updated